core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.controller.ts

```diff
@@ -104,15 +104,22 @@ angular.module('oppia').directive('topicLandingPage', [
           };
           ctrl.$onInit = function() {
             pathArray = WindowRef.nativeWindow.location.pathname.split('/');
-            ctrl.subject = pathArray[2];
-            topic = pathArray[3];
+            ctrl.subject = pathArray[1];
+            topic = pathArray[2];
             topicData = TOPIC_LANDING_PAGE_DATA[ctrl.subject][topic];
             landingPageData = topicData.page_data;
             ctrl.topicTitle = topicData.topic_title;
             ctrl.lessons = landingPageData.lessons;
             assetsPathFormat = '/landing/<subject>/<topic>/<file_name>';
-            var pageTitle = 'Learn ' + ctrl.topicTitle + ' - Oppia';
-            PageTitleService.setPageTitle(pageTitle);
+            var tagLines = {
```

Comment:
Should be in a constant somewhere higher up, right? E.g. perhaps at start of the controller -- I don't see why this only needs to be declared in onInit().

Reply:
Done.